src/beautifiers/exfmt.coffee

```diff
@@ -0,0 +1,36 @@
+###
+Requires https://github.com/lpil/exfmt
+###
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Exfmt extends Beautifier
+  name: "exfmt"
+  link: "https://github.com/lpil/exfmt"
+  executables: [
+    {
+      name: "exfmt"
+      cmd: "mix"
+      homepage: "https://github.com/lpil/exfmt"
+      installation: "https://github.com/lpil/exfmt"
+      additionalHelp: "If Elixir's mix is installed you can confirm exfmt is installed by running mix exfmt or install it with 'mix archive.install github lpil/exfmt tag v0.4.0'"
```

Comment:
Should info about `exfmt` be removed and only use the new and official `mix format` command?

Reply:
Yes please. exfmt is deprecated.